app/views/routes_into_teaching/steps/completed.html.erb

```diff
@@ -1,28 +1,105 @@
 <%= render Content::LandingHeroComponent.new(
-  title: "Routes into teaching",
+  title: "Find your route into teaching",
   colour: "pastel yellow-yellow",
 ) %>
 
 <section class="container">
-  <div class="row">
-    <h2>You told us that you:</h2>
+  <div class="row inset">
+    <div class="col col-845">
+      <h2>You told us that you:</h2>
 
-    <p>You can <%= link_to "change your answers", routes_into_teaching_steps_path %> if you need to.</p>
+      <ul>
+        <li><%= undergraduate_degree_summary %></li>
+        <li><%= unqualified_teacher_summary %></li>
+        <li><%= location_summary %></li>
+      </ul>
+
+      <% if non_uk? %>
+        <%= render Content::ExpanderComponent.new(
+          title: "visa sponsorship",
+          text: "<p>If you do not have the right to study or work in the UK, you will need a visa. Your visa will need to be sponsored by a course provider.</p>
+          <p><a href=\"/non-uk-teachers/visas-for-non-uk-trainees\">Learn more about applying for your visa to train to teach in England</a>.</p>",
+        ) %>
+      <% end %>
+
+      <p>You can <%= link_to "change your answers", routes_into_teaching_steps_path %> if you need to.</p>
+    </div>
   </div>
 
-  <div class="row">
-    <% @results.each do |result| %>
-      <div class="col-space-s">
-        <%= render Content::ResultsBoxComponent.new(
+  <div class="row inset">
+    <div class="col col-845">
+      <h2 class="heading--box-blue">Your options</h2>
+      <p>Based on your answers, you may be eligible for the following teacher training options:</p>
+    </div>
+  </div>
+
+  <div class="row inset">
+    <div class="col col-845">
+      <% @results.each do |result| %>
+        <div class="col-space-s">
+          <%= render Content::ResultsBoxComponent.new(
           heading: result["title"],
```

Comment:
@spencerldixon The route titles are all h2s but they should be h3s as they are under an h2 of `Your options`

![image](https://github.com/user-attachments/assets/89d222ed-ac4f-413a-859e-edbbee5054ec)